README.md

```diff
@@ -61,3 +61,16 @@ To release a new version of the Update Tool, run:
 This will release a stable version of whatever is indicated in the VERSION file. e.g. if VERSION contains `1.0.3-dev`, then version `1.0.3` will be tagged and released, and the VERSION file will be updated to `1.0.4-dev`. To release version `1.1.0` instead, manually edit the VERSION file to `1.1.0-dev` and then run `composer release`.
 
 The update-tool.phar file will be uploaded to GitHub on every release. Rebuild [pantheon-systems/docker-updatinator](https://github.com/pantheon-systems/docker-updatinator) to deploy a new version of the tool to the automation processes.
+
+## Project support levels
+
+| Badge | Description
+| ----- | -----------
+| [![Early Access](https://img.shields.io/badge/pantheon-EARLY_ACCESS-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/early-access?q=org%3Apantheon-systems)      | Early access, no support save from the Engineering team
```

Comment:
We can (and probably should) add alt text to all of these with the longer explanations/definitions. This would be great for a11y but also to add the contextual details that would be lost by the badge alone.

e.g.

`[![Early Access](https://img.shields.io/badge/pantheon-EARLY_ACCESS-yellow?logo=pantheon&color=FFDC28&style=for-the-badge)](https://github.com/topics/early-access?q=org%3Apantheon-systems "Early access, no support save from the Engineering team")`